index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: setup and mobile first</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/db2c8620f4.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li><img class="my-logo" src="Images_Background/Logo HG.png" alt="Personal Logo"></li>
+        <li><i class="fa-solid fa-bars hamburguer-menu"></i></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section id="introduction-section">
+      <h1 class="introduction-name">I’m Hernán.<br>Glad to see you!</h1>
+      <p class="introduction-paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <h2 class="lets-connect">LET'S CONNECT</h2>
+      <ul class="social-icons">
+        <li><a href="#"><img src="Images_Background/Twitter.svg" alt="Twitter Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/M.svg" alt="M Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/GitHub.svg" alt="GitHub Icon"></a></li>
+        <li><a href="#"><img src="Images_Background/Fingers.svg" alt="Fingers Icon"></a></li>
```

Comment:
- [OPTIONAL] I think it would be better if you add the links of your actual social profiles ✨ This is a great way to build a network. What do you think about it? 💭